python/src/scipp/compat/mantid.py

```diff
@@ -71,35 +71,28 @@ def make_variables_from_run_logs(ws):
     lookup_units.update(additional_unit_mapping)
     for property_name in ws.run().keys():
         units_string = ws.run()[property_name].units
-        units = lookup_units.get(units_string, None)
+        unit = lookup_units.get(units_string, sc.units.one)
         values = deepcopy(ws.run()[property_name].value)
 
-        if units_string and units is None:
+        if units_string and unit is None:
```

Comment:
With the new default of `sc.units.one` instead of `None` this check will not work any more?

Reply:
Thanks, missed that one, thankfully you and the unit tests didn't